angular/js/controllers/submitController.js

```diff
@@ -45,6 +45,7 @@ angular.module('nuswhispersApp.controllers')
     $scope.submitConfession = function () {
         $scope.form.submitButtonDisabled = true;
         $scope.confessionData.categories = $scope.form.selectedCategoryIDs;
+        $scope.confessionData.token = localStorageService.get('confessionToken');
```

Comment:
Hahaha I thought `token` should be generic enough to not be suspicious.

Good point! Will figure out how to take those keys out as constants.